k8s/production/runners/protected/graviton/2/release.yaml

```diff
@@ -185,7 +185,7 @@ spec:
       protected: true
       runUntagged: false
 
-      tags: "arm,aarch64,graviton,graviton2,small,medium,large,huge,protected,aws,spack"
+      tags: "graviton2,small,medium,large,huge,protected,aws,spack"
```

Comment:
shouldn't we remove this runner (and the corresponding public one) since we're removing its node pool?

Reply:
bleh, it got undeleted in a rebase, repushing